src/api/app/models/token/workflow.rb

```diff
@@ -29,10 +29,10 @@ def run_step_and_report(step, scm_extractor_payload, scm_token)
 
     set_subscription(package_from_step, scm_extractor_payload)
 
-    Project.get_by_name(step.target_project).repositories.each do |repository|
+    Project.get_by_name(step.target_project_name).repositories.each do |repository|
       # TODO: Fix n+1 queries
       repository.architectures.each do |architecture|
-        SCMStatusReporter.new({ project: step.target_project, package: step.target_package, repository: repository.name, arch: architecture.name },
+        SCMStatusReporter.new({ project: step.target_project_name, package: step.target_package, repository: repository.name, arch: architecture.name },
```

Comment:
One renaming is missing:

```suggestion
        SCMStatusReporter.new({ project: step.target_project_name, package: step.target_package_name, repository: repository.name, arch: architecture.name },
```

Reply:
done!